example/src/main/java/io/netty5/example/factorial/FactorialServer.java

```diff
@@ -43,7 +43,7 @@ public static void main(String[] args) throws Exception {
             ServerBootstrap b = new ServerBootstrap();
             b.group(bossGroup, workerGroup)
              .channel(NioServerSocketChannel.class)
-             .handler(new LoggingHandler(LogLevel.INFO))
+             .handler(new LoggingHandler(Level.INFO))
```

Comment:
I wonder if we want to have our own `Level` enum for the `LoggingHandler`. This way we at least not have it in the "public API" . WDYT ?